webhook/webhook_test.go

```diff
@@ -20,7 +20,7 @@ const (
 	apiSecret            = "mysecret"
 	testAddr             = ":8765"
 	testUrl              = "http://localhost:8765"
-	webhookCheckInterval = 100 * time.Millisecond
+	webhookCheckInterval = 500 * time.Millisecond
```

Comment:
did this change have any significance?

Reply:
changed it while testing to be sure everything was finishing, I'll change it back